src/trousse/feature_operations.py

```diff
@@ -25,6 +27,135 @@ def is_similar(self, other: "FeatureOperation"):
         raise NotImplementedError
 
 
+class _OperationsList:
+    def __init__(self):
+        self._operations_list = []
+        self._operations_by_column = collections.defaultdict(list)
+
+    def derived_columns_from_col(self, column: str) -> List[str]:
+        """Return name of the columns created by FeatureOperations applied on ``column``
+
+        Parameters
+        ----------
+        column : str
+            The column on which the FeatureOperation has been applied on.
+
+        Returns
+        -------
+        List[str]
+            Name of the columns created by FeatureOperations applied on ``column``.
+        """
+        derived_columns = []
+        operations = self[column]
+
+        for operation in operations:
+            if column in operation.columns and operation.derived_columns is not None:
```

Comment:
Similarly to the other comment, you should not end up having derived_columns = None?

Reply:
The "original" instances are stored, so I can have operations with derived columns = None.. but I cannot use the in operator in that case so I need an explicit check

Reply:
Right! I don't know what I read before :D